.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Are there any concerns here with this being invalid JSON?

Reply:
Ha, good question! This is a VS Code thing: they call it "JSONC" or "JSON with comments" but I guess they wanted to keep the `.json` file extension anyway. These comments are actually placed there by VS Code when you create the file.

See also: https://code.visualstudio.com/docs/languages/json#_json-with-comments